packages/material-ui-styles/src/withStyles/withStyles.d.ts

```diff
@@ -3,9 +3,15 @@ import { PropInjector } from '@material-ui/core';
 import * as CSS from 'csstype';
 import * as JSS from 'jss';
 
-export interface CSSProperties extends CSS.Properties<number | string> {
+export type CorrectedProperties<Props extends {} = {}> = {
```

Comment:
Can we improve the naming here? I don't understand what we're correcting here.

Reply:
`(Extended/Modified)Properties`?
We're correcting(modifying? extending?) the default `CSS.Properties<number | string>` to allow a value to be a function that returns a value